lldb/include/lldb/Utility/Log.h

```diff
@@ -272,6 +272,12 @@ class Log final {
   void VAFormatf(llvm::StringRef file, llvm::StringRef function,
                  const char *format, va_list args);
 
+  void Enable(const std::shared_ptr<LogHandler> &handler_sp,
+              uint32_t options = 0,
+              MaskType flags = std::numeric_limits<Log::MaskType>::max());
```

Comment:
The log channels have a concept of a default set of categories, so if we're going to have a default here, it better be it. That could be achieved by making this an `optional<MaskType>` (and then you can also simplify the code in `EnableLogChannel`).